### ISSUE TITLE
Why the training time is so long?

### ISSUE THREAD

**Issue by nickucas123** [mrakotosaon/pointcleannet#10]
Hello, your code about denoising, when the epoches is  2000, the training time is about 66 hours. The graphics card I use is rtx3080. Why is the training time so long,How long did you use it when training denoising？

**Comment by mrakotosaon** [comment]
Hi @nickucas123 

The pre-trained model that we give was trained for a long time (however we did stop the training earlier than 2000 epochs).  It is possible to lower the training time by changing the training parameters (larger learning rate) and stopping the training early. This would depend on your training data. For training our models, we referred to the training curves to determine when to stop training.


Best,
